scripts/patterns-reorg/page_content.py

```diff
@@ -14,69 +14,126 @@
 
 
 def map_content(index: str) -> str:
-    return (
-        "# Map problem to circuits\n\n"
-        'The "map problem to circuits" step of a Qiskit pattern describes how a user starts with a classical '
-        "problem and figures out how to map it to a quantum computer. For example, in applications such as "
-        "chemistry and quantum simulation, this step generally involves constructing a quantum circuit "
-        "representing the Hamiltonian you are attempting to solve. During this step, for certain problems, "
-        "it may also be desirable to specify the mapping of the problem onto qubits in the heavy-hex (or "
-        "gross) lattice of IBM Hardware from the outset if the structure of the problem lends itself to "
-        "optimization earlier. It is also worth considering at this point what the outcome of the particular "
-        "algorithm will be in prepartion for the later execute step, e.g. if the desired outcome involves "
-        "inferring correlation functions using Hadamard tests, you might prepare to use `Sampler` whereas "
-        "specifying observables would use the `Estimator` and could provide many error mitigation options. "
-        "The output of this step in a Qiskit pattern is normally a collection of circuits or quantum operators.\n\n"
-        "There are a number of software tools that could be used for this step, you can check out the "
-        f"following resources:\n\n{index}\n"
+    intro = dedent(
+        """\
+---
+title: Map problem to circuits
+description: TODO - 50+ characters long to ignore metadata check for now!
+---
+
+# Map problem to circuits
+
+The "map problem to circuits" step of a Qiskit pattern describes how a user starts with a classical
+problem and figures out how to map it to a quantum computer.
+
+For example, in applications such as chemistry and quantum simulation, this step generally involves 
+constructing a quantum circuit representing the Hamiltonian you are attempting to solve.
+During this step, for certain problems, it may also be desirable to specify the mapping of
+the problem onto qubits in the heavy-hex (or gross) lattice of IBM Hardware from the
+outset if the structure of the problem lends itself to optimization earlier. 
+
+It is also worth considering at this point what the outcome of the particular algorithm will be
+in prepartion for the later execute step, e.g. if the desired outcome involves
+inferring correlation functions using Hadamard tests, you might prepare to use `Sampler` whereas
+specifying observables would use the `Estimator` and could provide many error mitigation options.
+
+The output of this step in a Qiskit pattern is normally a collection of circuits or quantum operators.
+
+## Guides
+"""
     )
+    return f"{intro}{index}\n"
 
 
 def postprocess_index_content(index: str) -> str:
-    return (
-        "# Post-process results\n\n"
```

Comment:
I see that all the other sections preserve the title except this one. Is this intentional?

Reply:
Oh, oops. Good catch!